### ISSUE TITLE
My Cars compatibility

### ISSUE THREAD

**Issue by sanmibuh** [Programistyk/FuelioImport#12]
Please add compatibility with "my Cars"

Best

**Comment by viharm** [comment]
Yes, please add compatibility with [My Cars](https://play.google.com/store/apps/details?id=com.aguirre.android.mycar.activity).

**Comment by britshickhouse** [comment]
Hi @mrblur , another vote for My Cars, and I'm happy to pay for a My Cars Pro licence if you need it for development.
